gammapy/irf/effective_area_table.py

```diff
@@ -444,6 +474,45 @@ def read(filename):
         hdu_list = fits.open(filename)
         return EffectiveAreaTable2D.from_fits(hdu_list)
 
+    def to_effective_area_table(self, offset, energ_lo=None, energ_hi=None):
+        """Evaluate at a given offset and return effective area table.
+
+        The energy thresholds in the effective area table object are not set.
+        If the effective area table is intended to be used for spectral analysis,
+        the final energy binning should be given here, since the
+        effective area table class does no interpolation.
+
+        Parameters
+        ----------
+        offset : `~astropy.coordinates.Angle`
+            offset
+        energy : `~astropy.units.Quantity`
```

Comment:
Parameter docs (`energy`) is inconsistent with actual parameters (`energ_lo` and `energhi`).

I don't like `energ` ... can you use `energy` in Gammapy for variable names even if some FITS format uses `energ`?